asapdiscovery-data/asapdiscovery/data/fitness.py

```diff
@@ -140,35 +148,48 @@ def parse_fitness_json(target: TargetTags) -> pd.DataFrame:
             f"Specified target is not valid, must be one of: {TargetTags.get_values()}"
         )
 
-    if target not in ("SARS-CoV-2-Mpro", "SARS-CoV-2-Mac1"):
+    if target not in ("SARS-CoV-2-Mpro", "SARS-CoV-2-Mac1", "ZIKV-NS2B-NS3pro"):
         raise NotImplementedError(
             f"Fitness data not yet available for {target}. Add to metadata if/when available."
         )
 
-    with open(SARS_CoV_2_fitness_data) as f:
-        data = json.load(f)
-    data = data["data"]
-    fitness_scores_bloom = pd.DataFrame(data)
-    # now get the target-specific entries.
-    fitness_scores_bloom = fitness_scores_bloom[
-        fitness_scores_bloom["gene"] == _TARGET_TO_GENE[target]
-    ]
-    if target == "SARS-CoV-2-Mac1":
-        # need to subselect from nsp3 multidomain to get just Mac1. See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7113668/
-        fitness_scores_bloom = fitness_scores_bloom[
-            fitness_scores_bloom["site"].between(209, 372)
-        ]
-        fitness_scores_bloom[
-            "site"
-        ] -= 208  # reindex to set residue numbers to correct values
-    elif target == "SARS-CoV-2-Mpro":
-        # simpler; can just query the correct gene in the JSON.
+    ## START TODO
+    if "SARS-CoV-2" in target:  # TODO: generalize this block
+        threshold = -1.0
+        with open(SARS_CoV_2_fitness_data) as f:
+            data = json.load(f)
+        data = data["data"]
+        fitness_scores_bloom = pd.DataFrame(data)
+
+        # now get the target-specific entries. Need to do because the phylo data is cross-genome.
         fitness_scores_bloom = fitness_scores_bloom[
-            fitness_scores_bloom["gene"] == "nsp5 (Mpro)"
+            fitness_scores_bloom["gene"] == _TARGET_TO_GENE[target]
         ]
 
+        if target == "SARS-CoV-2-Mac1":
+            # need to subselect from nsp3 multidomain to get just Mac1. See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7113668/
+            fitness_scores_bloom = fitness_scores_bloom[
+                fitness_scores_bloom["site"].between(209, 372)
+            ]
+            fitness_scores_bloom[
+                "site"
+            ] -= 208  # reindex to set residue numbers to correct values
+        elif target == "SARS-CoV-2-Mpro":
+            # simpler; can just query the correct gene in the JSON.
+            fitness_scores_bloom = fitness_scores_bloom[
+                fitness_scores_bloom["gene"] == "nsp5 (Mpro)"
+            ]
+    # for ZIKV NS2B3 it's simpler - no need to subselect the target from the genome.
+    elif target == "ZIKV-NS2B-NS3pro":
+        threshold = 0.0
```

Comment:
Current thresh 0.0 for ZIKV?

Reply:
yep, it's a completely different experiment so the dynamic range is different. For this DMS analysis, values are a log transformation from 0: a value of -1 means 10x less fit than wildtype, a value of 2 means 100x more fit than mutant.

I'm double-checking with Jesse Bloom to make sure I took the correct score here but I'm fairly confident, so we can merge imo

Reply:
issue raised #654